apps/vor/pkg/postgres/postgres.go

```diff
@@ -244,15 +244,22 @@ type Weekday struct {
 }
 
 type (
+	LessonPlanDetails struct {
+		Id                string `pg:"type:uuid"`
```

Comment:
why title and description is nullable?

Reply:
Hmmm goodpoint, those should probably be non-pointer.

Those are seperated to make it possible to share the same details across multiple lesson plan instances, such as when there are repetitions.

Reply:
pointer is no more now